### ISSUE TITLE
[Dev Site] Update all instrumentation to use tessen

### ISSUE THREAD

**Issue by LizBaker** [newrelic/developer-website#1885]
This ticket is blocked by https://github.com/newrelic/gatsby-theme-newrelic/issues/559

### Acceptance Criteria

`note` 1st, verify if the site has any browser agent page actions that were not covered by the theme changes. 

- [x] Bring in new version of the theme
- [x] remove all browser agent page actions from the site
- [x] Check all instances of the `instrumentation` prop and make sure it follows the correct tessen formatting (category is Title Case, event name is camelCase)
- [x] Check for any uses of the `useInstrumentedHandler` hook and make sure it’s formatted correctly for tessen
- [x] Check for any uses of the browser agent directly (window.newrelic.pageAction)
- [x] Update any documentation/wiki references to the browser events
- [ ] Update any dashboards that may be using the old browser events (events should be listed in the wiki but may not be up to date. You can also run [this query](https://staging.onenr.io/0a2wdAvxpRE) to get a list of currently firing browser actions ((some may be coming from theme components))

Here are some important dashboards to look out for:
* [DevEn] External I/O & Quickstarts Ecosystem
* New Relic I/O Program Dashboard
* [DevEn] Docs Ecosystem

**Comment by nr-kkenney** [comment]
docs ecosystem dashboard updates live in another dashboard -- `docs ecosystem - tessen only`. atm, many of the charts are empty due to docs website not being updated yet (i assume). i learned on friday that we could make all the changes we wanted w/o (potentially) impacting our charts because we still have page events going to our account when we use tessen. however, the charts in the ecosystem dashboard use the dev ex account rather than dev enablement, so we can't have data sources in two accounts.

tl;dr -- there are two dashboards atm. the old one which is mostly unmodified, and the new one which looks at data in the dev enablement and tessen accounts.
